trunk.hcl

```diff
@@ -0,0 +1,11 @@
+description = "Trunk is a CI runner."
+binaries = [ "trunk" ]
+test = "trunk --version"
+
+linux {
+  source = "https://trunk.io/releases/${version}/trunk-${version}-linux-x86_64.tar.gz"
+}
+
+darwin {
+  source = "https://trunk.io/releases/${version}/trunk-${version}-darwin-x86_64.tar.gz"
+}
```

Comment:
Need to add a `version` block here:

```hcl
version "1.9.1" { }
```

Reply:
Ahh that makes sense. You just wanted me to remove the auto-version update.